eslint/eslint-bulk/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@rushstack/eslint-bulk",
+  "version": "1.5.1",
```

```suggestion
  "version": "0.0.0",
```

Set this to 0.0.0 so the initial (minor) release will be 0.1.0

---

(Alternatively, if we wanted to share the same version as `eslint-patch` then we would need to use a Rush a lockstep version policy; simply setting the versions equal is not enough, because they will quickly drift out of sync.)

---

I changed it to 0.0.0, though maybe its ready for 0.1.0

---

It will get bumped to 0.1.0 automatically during publishing (because your change file says `minor`)